tests/dummy/app/task-injection-test/controller.js

```diff
@@ -7,18 +7,17 @@ export default Controller.extend({
 
   myTask: task({
     fun: service(),
-    perform: function * () {
+    *perform() {
```

Comment:
seems unrelated?

Reply:
unrelated to the fix but _cleaner_. Didn't feel like it warranted omission 😛